lib/bundle/brew_installer.rb

```diff
@@ -75,6 +75,11 @@ def link_change_state!
       when false
         puts "Unlinking #{@name} formula." if ARGV.verbose?
         Bundle.system("brew", "unlink", @name)
+      when nil
+        if unlinked?
```

Comment:
Does this verify keg-only state?

Reply:
Yes.
https://github.com/Homebrew/homebrew-bundle/blob/d6e194364227dfdc11dff12d86d1dfee7b065f5f/lib/bundle/brew_dumper.rb#L119-L126

Reply:
I renamed this predicate `unlinked?` to `unlinked_and_not_keg_only?`.